pages/_language/events/_event/index.vue

```diff
@@ -23,46 +27,65 @@
           height="35"
           alt=""
         >
-        <h3 class="event-section__title">
-          {{ section.title }}
-        </h3>
-        <img
-          v-if="section.image"
-          class="event-section__image"
-          :src="`${section.image.url}?auto=compress`"
-          :alt="section.image.alt"
-          width="320"
-        >
-        <span
-          class="event-section__body"
-          v-html="section.body"
-        />
-        <ul class="event__relevant-chapters">
-          <li
-            class="chapter-preview"
-            v-for="chapter in section.chapters"
-            :key="chapter.slug"
+        <section class="event__layout event__layout--padded event-section__body">
+          <img
+            v-if="section.waveMarker"
+            class="event-section__divider"
+            src="/event-section-wave.svg"
+            width="111"
+            height="35"
+            alt=""
+          >
+          <h3
+            class="event-section__title"
+            :class="{
+              'event-section__title--orange': section.titleColor === 'orange',
+              'event-section__title--blue': section.titleColor === 'blue',
+            }"
+          >
+            {{ section.title }}
+          </h3>
+          <img
+            v-if="section.image"
+            class="event-section__image"
+            :src="`${section.image.url}?auto=compress`"
+            :alt="section.image.alt"
+            width="320"
           >
-            <nuxt-link
-              class="chapter-preview__link"
-              :to="chapter.slug"
-              append
+          <span
+            class="event-section__copy"
+            v-html="section.body"
+          />
+          <ul
```

Comment:
Unrelated to the text block ticket but already had these chapter preview changes locally. Since the diff was hard to split I included it.